src/client/core.ts

```diff
@@ -88,7 +83,7 @@ export async function aptosRequest<Req, Res>(
       throw new AptosApiError(
         options,
         result,
-        response.data.errors[0].message ?? `Unhandled Error ${response.status} : ${response.statusText}`,
+        (result.data as any).errors[0].message ?? `Unhandled Error ${response.status} : ${response.statusText}`,
```

Comment:
This may be a good time to type this output or, type guard it so result.data is any so we don't keep redoing it over and over below.

e.g.
```
const data = result.data as any;

if (data.errors) {
  throw new data.errors[0].message ...
}
```
etc.

Either that, or type it if possible, but I guess it's kinda the any randomness.

Interesting that it doesn't need `?` before each sub item.